client/src/index.css

```diff
@@ -1,3 +1,5 @@
+/* https://coolors.co/cfe5f8-a8c9ec-7099d2-01161e-f56476 */
```

Comment:
At some point later we'll have to go through a color pallets with our client and check with the accessibility guidelines.
See: https://refactoringui.com/previews/building-your-color-palette/ & https://commandcenter.blogspot.com/2020/09/color-blindness-is-inaccurate-term.html & https://www.toptal.com/designers/colorfilter but I like the current design it looks nice.